# --- T2-COPYRIGHT-NOTE-BEGIN ---
# T2 SDE: package/*/firefox/hotfix-fdlibm.patch.x86
# Copyright (C) 2021 The T2 SDE Project
# 
# This Copyright note is generated by scripts/Create-CopyPatch,
# more information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License version 2 as used by the T2 SDE.
# --- T2-COPYRIGHT-NOTE-END ---

--- firefox-95.0.2/modules/fdlibm/src/math_private.h.vanilla	2021-12-28 17:23:16.977090511 +0100
+++ firefox-95.0.2/modules/fdlibm/src/math_private.h	2021-12-30 13:10:59.191804064 +0100
@@ -30,7 +30,7 @@
  * Adapted from https://github.com/freebsd/freebsd-src/search?q=__double_t
  */
 
-typedef double      __double_t;
+typedef long double      __double_t;
 typedef __double_t  double_t;
 
 /*
